Fixes #26128 - pin highlight.js package

amirfefer opened the pull request:
npm install ends with an error:
```
npm ERR! code ENOENT
npm ERR! errno -2
npm ERR! syscall chmod
npm ERR! enoent ENOENT: no such file or directory, chmod '/home/afeferku/git/foreman/node_modules/highlight.js/tools/build.js'
```

caused by hightlight.js package:
https://github.com/highlightjs/highlight.js/issues/1984

it seems that version`9.14` is the latest stable

ekohl commented:
A note in general: it helps if you write in the body why it was pinned. In my experience this helps when figuring out if it can be unpinned. Github also automatically copies the body of the commit to the PR message.

amirfefer committed:
Fixes #26128 - pin highlight.js package

highlight.js v.9.15+ causes an error after running
npm install, for more details see issue #1984
in highlight.js repo

amirfefer commented:
Thanks @ekohl , makes sense,  I've added the reason for this inside the commit.

waldenraines reviewed (approved):
This change fixes this for me.  Approved pending adding a comment about why this was pinned.

waldenraines commented:
> Thanks @ekohl , makes sense, I've added the reason for this inside the commit.

I think he means inside the file itself.

ekohl reviewed (approved):
> I think he means inside the file itself.

JSON files can't have comments. A commit body is fine because git blame tells you enough. Generally it's best if you don't need to refer to an external source and just inline the entire description but adding an additional link can be useful.

waldenraines commented:
> JSON files can't have comments.

Lol, I guess I needed more coffee when I made that comment.  :+1:

ohadlevy commented:
[test foreman]

ohadlevy commented:
test failures doesnt seem related, thanks
